website/server/controllers/api-v3/members.js

```diff
@@ -417,8 +417,9 @@ api.sendPrivateMessage = {
     let userBlockedSender = receiver.inbox.blocks.indexOf(sender._id) !== -1;
     let userIsBlockBySender = sender.inbox.blocks.indexOf(receiver._id) !== -1;
     let userOptedOutOfMessaging = receiver.inbox.optOut;
+    let senderIsNotAdmin = !sender.contributor.admin;
```

Comment:
Can you change this to `Boolean(sender.contributor.admin)`. I think that will be safer and more readable.